pkg/foxtrot/db.go

```diff
@@ -53,18 +54,96 @@ func (db *db) setupSchema() error {
 
 func (db *db) getUser(ctx context.Context, name string) (*User, error) {
 	u := User{Name: name}
-	selectUserStmt := "SELECT password_hash FROM users WHERE name=?"
-	err := db.conn.QueryRowContext(ctx, selectUserStmt, name).Scan(&u.passwordHash)
+	stmt := "SELECT password_hash FROM users WHERE name=?"
+	err := db.conn.QueryRowContext(ctx, stmt, name).Scan(&u.passwordHash)
 	if err != nil {
 		return nil, fmt.Errorf("db: cannot get user '%s': %w", name, err)
 	}
 	return &u, nil
 }
 
 func (db *db) createUser(ctx context.Context, u *User) error {
-	insertUserStr := "INSERT INTO users(name, password_hash, avatar) VALUES( ?, ?, ? )"
-	if _, err := db.conn.ExecContext(ctx, insertUserStr, u.Name, u.passwordHash, u.avatar); err != nil {
+	stmt := "INSERT INTO users(name, password_hash, avatar) VALUES( ?, ?, ? )"
+	if _, err := db.conn.ExecContext(ctx, stmt, u.Name, u.passwordHash, u.avatar); err != nil {
 		return fmt.Errorf("db: cannot create user '%s': %w", u.Name, err)
 	}
 	return nil
 }
+
+func (db *db) getRoom(ctx context.Context, name string) (*Room, error) {
+	r := Room{Name: name}
+	stmt := "SELECT name FROM rooms WHERE name=?"
```

Comment:
nit: the getMessages sql has spaces around the `=` (and `<`), but I note that getUser does not

Reply:
fixed.